### DISCUSSION TITLE
Is this technically a full replacementt for CrewChief?

### DISCUSSION THREAD

**Discussion by VFXpro** [SeriousOldMan/Simulator-Controller#18] [General]
Before I invest time learning this great app, is it considered a replacement for Crew Chief or it runs side by side as a complement?  There seems to be overlap in functionality, thus why I wonder...

**Comment by SeriousOldMan** [comment]
You can do both. My software is fully configurable and you can disable the Spotter to use CC here, for example. But my software delivers almost all that CC offers and so much more. To be honest the voice quality in CC is a little bit better, since CC uses recorded voices and my software implements a synthesized approach. If you use neural voice  synthesis from the Azure cloud, the quuality is almost the same, though.

Be aware, installation and configuration is not easy, but it is worth it. If you struggle, visit our Discord and let me know.
